### PR TITLE
Add more Rewards QA

### PR DESCRIPTION
**Pull request by piazzatron**
### Description

Still need to do Discord drawer, will be in the other PR. 

### Dragons

- Lineup hacky fix :( done with @jowlee 

### How Has This Been Tested?

Locally

### CODE REVIEW

File: src/containers/artist-dashboard-page/ArtistDashboardPage.tsx [hunk 1]
```diff
@@ -246,6 +246,12 @@ export class ArtistDashboardPage extends Component<
     TotalPlaysChart.preload()
   }
 
+  componentDidUpdate() {
+    if (!this.props.account?.is_creator) {
+      this.props.goToRoute(TRENDING_PAGE)
```
Comment (raymondjacobson) [inline src/containers/artist-dashboard-page/ArtistDashboardPage.tsx:251]: is there a possibility this runs a ton of times before unmount?

File: src/containers/trending-playlists/store/lineups/collections/reducer.ts [hunk 2]
```diff
@@ -4,7 +4,8 @@ import { PREFIX } from './actions'
 
 export const initialState = {
   ...initialLineupState,
-  prefix: PREFIX
+  prefix: PREFIX,
+  maxEntries: 30
```
Comment (raymondjacobson) [inline src/containers/trending-playlists/store/lineups/collections/reducer.ts:8]: why is this 30? is that what the backend does? maybe add a comment for when we rework this

**Review by raymondjacobson — APPROVED** [review]

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/piazz-rewards-qa-4

File: src/containers/trending-playlists/store/lineups/collections/reducer.ts (hunk 2 above)
Comment (piazzatron) [reply src/containers/trending-playlists/store/lineups/collections/reducer.ts:8]: Yeah, playlist trending is limited to 30 on the backend for perf reasons

File: src/containers/artist-dashboard-page/ArtistDashboardPage.tsx (hunk 1 above)
Comment (piazzatron) [reply src/containers/artist-dashboard-page/ArtistDashboardPage.tsx:251]: Theoretically I think it could, but I didn't see anything weird in my testing

**Pull request merged**